.nvmrc

```diff
@@ -1 +1 @@
-16
+18.14.2
```

Comment:
Upgraded to 18 not 20 as 20 is not yet supported in Jenkins by DevOps. tey have plans to support it in Q1 next year